accessibility/alt-text.md

```diff
@@ -167,4 +170,28 @@ However, you MUST be careful with this - not everyone is comfortable with having
 * [Writing great alt text: Emotion matters](https://jakearchibald.com/2021/great-alt-text/)
 * [Thoughts on skin tone and text descriptions](https://tink.uk/thoughts-on-skin-tone-and-text-descriptions.md-notes-on-synthetic-speech/), by Léonie Watson
 
+### Images of text 
 
+<img width="625" alt="screenshot of social media post of an image of text" src="https://github.com/springernature/frontend-playbook/assets/69847167/83772c8e-6b61-4f5a-8970-fc91be79f84d">
```

Comment:
alt text examples go here: https://github.com/springernature/frontend-playbook/tree/main/accessibility/images/alt-examples (there's an upload button top right on that page, if you're using the web interface on Github), then your src will be `src="images/alt-examples/orlok.png"` (or whatever file name you're using)

Reply:
will change on another PR